src/access/workflows/LendAndBridgeWorkflow.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.18;
+
+import {IERC20} from "@openzeppelin-5.0.1/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin-5.0.1/contracts/token/ERC20/utils/SafeERC20.sol";
+import {Address} from "@openzeppelin-5.0.1/contracts/utils/Address.sol";
+import {IERC4626} from "@openzeppelin-5.0.1/contracts/interfaces/IERC4626.sol";
+
+import {IAavePool, IPoolAddressesProvider, IStaticATokenFactory} from "@kinto-core/interfaces/external/IAavePool.sol";
+import {IBridge} from "@kinto-core/interfaces/bridger/IBridge.sol";
+import {IAccessPoint} from "@kinto-core/interfaces/IAccessPoint.sol";
+import {IBridger} from "@kinto-core/interfaces/bridger/IBridger.sol";
+
+contract LendAndBridgeWorkflow {
```

Comment:
This is not needed anymore right? We can remove this workflow

Reply:
Yes. This is not needed.